plugins/automation/examples/development-namespace.yaml

```diff
@@ -0,0 +1,10 @@
+name: "Do not scan development namespace for Admission Controller"
+description: "Mark any action items in the development namespace for the Admission Controller context as working as intended"
```

Comment:
I think this description might need to change. We could change it to be something like "Don't fail Admission Controller for the Development Namespace" something like that but better :)

Reply:
Updated. What do you think?